sw/device/sca/otbn_vertical/ecc256_keygen_serial.c

```diff
@@ -234,6 +270,90 @@ void ecc256_ecdsa_secret_keygen_batch(const uint8_t *data, size_t data_len) {
                             kEcc256SeedNumWords * 4);
 }
 
+void add_arrays(uint8_t *dest, uint8_t *source, size_t dest_len,
```

Comment:
This function should be declared as `static` and it needs  a comment header explaining what the arguments are and what the function does.

Reply:
fixed